QSB/PlayerRegistry.cs

```diff
@@ -1,149 +1,46 @@
-﻿using OWML.Common;
-using QSB.Animation;
-using QSB.Events;
-using QSB.Utility;
+﻿using QSB.Events;
 using System.Collections.Generic;
 using System.Linq;
-using UnityEngine;
+using QSB.TransformSync;
 
 namespace QSB
 {
     public static class PlayerRegistry
     {
-        public static List<PlayerInfo> PlayerList { get; } = new List<PlayerInfo>();
+        public static PlayerInfo LocalPlayer => GetPlayer(PlayerTransformSync.LocalInstance.netId.Value);
```

Comment:
does this run as soon as the script is loaded? because this will fail then

Reply:
nope, when it's called